.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "ocaml.sandbox": {
+        "kind": "custom",
+        "template": "cd ${firstWorkspaceFolder} && fnm exec --using $(cat .nvmrc) -- npx esy $prog $args"
```

Comment:
As in the updates to the Makefile, here we also assume that developers are using `esy`. We need to review with @pveber. If he uses VS Code, this would affect him too.

Reply:
This is now moot since we decided to remove the VS Code configuration.